src/app/zap-templates/zcl/data-model/chip/dishwasher-alarm-cluster.xml

```diff
@@ -60,9 +60,4 @@ limitations under the License.
         <field id="3" name="Mask" type="AlarmBitmap" />
     </event>
   </cluster>
-
-  <bitmap name="Feature" type="bitmap32">
-    <cluster code="0x005D" />
-    <field name="Reset" mask="0x01" />
-  </bitmap>
```

Comment:
No features for this in new xml

Reply:
Has this shipped anywhere?  If so, it needs to stay, however we accomplish it.  I agree this is not in the spec right now, but clearly it was at some point, since the XML is here.  When was it removed, and why was the XML not updated?

Reply:
Added it back

Reply:
To answer my own question: This is a derived cluster, and the feature is on the derived-from cluster.  The spec scraper is not understanding that structure, apparently.